### PR TITLE
manually add tocbot w/ npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
@Amomo @Piicksarn could u help me check this PR for #8844 ? thank you!

**the author has added npm auto-update for us, and he will put dist to npm package in next version.
I didn't add map files for not minified files to keep the file structure.**

**git repo url:** https://github.com/tscanlin/tocbot
**Watch 10
  Star 110
 Fork 9**

**npm package url:** https://www.npmjs.com/package/tocbot
**66 downloads in the last day
122 downloads in the last week
380 downloads in the last month**

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR 
 * [x] The lib has notable popularity  
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry 
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update) valid from 2.1.5
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/tocbot/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  "author": "Tim Scanlin",
+  "license": "MIT",
+  "dependencies": {
+    "smooth-scroll": "cferdinandi/smooth-scroll"
+  },
+  "npmName": "tocbot",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```
Comment (PeterDaveHello) [inline ajax/libs/tocbot/package.json]: I didn't see `dist` folder in its npm package.

**Comment by PeterDaveHello** [comment]
What's the source of the css file in v1.0?

**Comment by maruilian11** [comment]
https://github.com/tscanlin/tocbot/releases/download/v1.0/style.css
https://github.com/tscanlin/tocbot/releases/download/v1.0/tocbot.js

**Comment by maruilian11** [comment]
I ask the author if he can release new version recently, and he said he don't have any changes to be able to release a new version right now. 
https://github.com/tscanlin/tocbot/issues/16#issuecomment-247370522

can we still add auto-update config in this PR? or I can remove the config and wait the new version.

**Comment by maruilian11** [comment]
ping @Piicksarn @Amomo

**Comment by Piicksarn** [comment]
@maruilian11, 
i think **adding the auto-update is better**, and git auto-update is better than npm auto-update, cause the structure with dist folder only on github

**Comment by PeterDaveHello** [comment]
npm auto-update is fine since author said `dist` will be published there in future release. css name in v0.0.1 should be modified as other versions.

**Comment by maruilian11** [comment]
@PeterDaveHello I think you mean v1.0, and I have modified .css name.
@Piicksarn @Amomo could you help me check this PR? thanks!

**Pull request merged**